### DISCUSSION TITLE
App Store [Nov 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4443] [Discord Threads]
> 🙋‍♂️ **amwnd**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Project menu > Team & Settings > App Store  
#  
> 👨‍🦲 **dstala**  
#### Click on Project name on left top to access `Project menu`  
#  
> 👨‍🦲 **dstala**  
#### what `role` you are logged in as?  
#  
> 🙋‍♂️ **amwnd**  
#### objects.roleType.owner,creator  
#  
> 👨‍🦲 **dstala**  
#### Only `Super Admin` will have AppStore access. This was changed in one of recent release.  
#  
> 🙋‍♂️ **amwnd**  
#### I am the only user on the server so shouldnt i be the superadmin  
#  
> 🙋‍♂️ **amwnd**  
#### Or do i need to reinstall?  
#  
> 👨‍🦲 **dstala**  
#### only user, then you should have access. if you can, try reinstall if that helps. if issue persists, let us know  
#  
> 🙋‍♂️ **amwnd**  
#### Had to reinstall- new DB, now it works..  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1035463463864119296)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by Charuru** [comment]
Hmm I can't get this working, new install, only user.

**Reply by wingkwong** [reply]
Please share more details. App Store is located in Account Settings and only for super admin.

See https://docs.nocodb.com/setup-and-usages/account-settings#app-store

**Reply by Charuru** [reply]
Thanks, I went to team and settings as per the thread above and couldn't find it, but I see it in account settings now.
